packages/server/src/plugin/usageReporting/plugin.ts

```diff
@@ -737,8 +737,10 @@ export function ApolloServerPluginUsageReporting<TContext extends BaseContext>(
               asTrace:
                 sendTraces &&
                 (!isExecutable || !!metrics.captureTraces) &&
-                sendOperationAsTrace(trace, statsReportKey),
+                sendOperationAsTrace(trace, statsReportKey) &&
+                !metrics.nonFtv1ErrorPaths?.length,
```

Comment:
(a) this probably deserves a comment (or an expansion of the larger comment above, which maybe could be a bulleted list by now)
(b) might as well do this boolean check before calling the hook on the previous line instead of after?

Reply:
[5c19b5a](https://github.com/apollographql/apollo-server/pull/7136/commits/5c19b5ad6e4e15e2bedb9d86514bb4fce81bb8ec)